CMakeLists.txt

```diff
@@ -439,6 +460,10 @@ if (WIN32)
   target_compile_definitions (glog PUBLIC GLOG_NO_ABBREVIATED_SEVERITIES)
 endif (WIN32)
 
+if (MSVC)
+  target_compile_definitions (glog PRIVATE _SCL_SECURE_NO_WARNINGS)
```

Comment:
I didn't particularly want to add this, as it's disabling valid warnings, where the code itself probably should be fixed (at some point). I chose not to hide them. Do you think we should do this, and if so, should we add at least a TODO to fix the source of the warnings?